task_12/style/sass/buttons.sass

```diff
@@ -0,0 +1,59 @@
+.button 
+    padding: 1em 2em 
+    text-transform: uppercase 
+    font-weight: 700
+    background: none
+    border: none
+    cursor: pointer
+    box-shadow: 0 1px 3px rgba(0,0,0,0.12), 0 1px 2px rgba(0,0,0,0.24)
+    transition: all 0.3s cubic-bezier(.25,.8,.25,1)
+    position: relative
+    i 
+        margin: 0 0.5em 0 0
+    &:hover
+        box-shadow: 0 14px 28px rgba(0,0,0,0.25), 0 10px 10px rgba(0,0,0,0.22)
+    &__dark-orange 
+        background: #ea5800
+        color: #fff
+    &__dark-green 
+        background: #067500
+        color: #fff
+    &__stripe
+        overflow: hidden
+        color: #fff
+        &:before
+            background: linear-gradient(to right, #067500 0%, #067500 50%, #034000 50%, #034000 100%)
+            content: ''
+            position: absolute
+            top: 0
+            left: 0
+            width: 200% 
+            height: 100% 
+            z-index: -2
+            transition: .3s ease-in-out
+        &:after
+            content: '' 
+            width: 0 
+            height: 0 
+            border-style: solid 
+            border-width: 0 0 80px 80px 
+            border-color: transparent transparent #034000 transparent 
+            position: absolute
+            right: -5px 
+            top: -5px 
+            z-index: -1 
+            transition: .3s ease-in-out
+        &:hover
+            color: #fff 
+            &:before
```

Comment:
Try to avoid nesting more than 3 levels deep (including root level).

Always use `::` (double colon) with pseudo elements and `:` (single colon) with pseudo classes. It's the new syntax.